bundles/automation/org.eclipse.smarthome.automation.core/OSGI-INF/RuleEngine.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please replace this xml file with the according DS annotation in the code so this file will be auto generated? You can then also add this file to the .gitignore file in the `OSGI-INF` folder.